proto/pulumi/provider.proto

```diff
@@ -65,6 +66,8 @@ service ResourceProvider {
 
     // GetSchema fetches the schema for this resource provider.
     rpc GetSchema(GetSchemaRequest) returns (GetSchemaResponse) {}
+    // GetPackageInfo tries to find the package info for the given package and version.
+    rpc GetPackageInfo(GetSchemaRequest) returns (pulumirpc.codegen.PackageInfo) {}
```

Comment:
I ended up leaving this in the ResourceProvider service because that seems simpler.  Happy to be convinced otherwise.

Reply:
Initial bikeshed is whether we should go `GetSchemaPackageInfo`, `GetSchemaResource`, since we'll have a few and I could see e.g. `GetResource` being maybe confusing, etc. but no strong opinions.

Reply:
I think that's a good idea if we're putting these directly in the provider service. If we pull this into it's own service (and we should sync with providers team about pros/cons of that) then names can be more succinct.